include/language-factory.php

```diff
@@ -41,18 +41,18 @@ public static function get( $language_data ) {
 	 *
 	 * @since 3.4
 	 *
-	 * @param WP_Term[] $terms List of language terms, with the type as array keys.
-	 *                         `post` and `term` are mandatory keys.
+	 * @param WP_Term[] $terms List of language terms, with the language taxonomies language as array keys.
```

Comment:
```suggestion
	 * @param WP_Term[] $terms List of language terms, with language taxonomy names as array keys.
```
Just suggest a more descriptive sentence.

Reply:
Done in cbe5d977df3438ec872accd81e82eb4604e390cd